main.go

```diff
@@ -170,13 +170,48 @@ func getBasicInfoOfType(c echo.Context) error {
 	var agencies []strModels.Agency
 	var err error
 	var estadual bool
-	// Verificando se trata-se de um estado ou jurisdição
-	values := [27]string{"AC", "AL", "AP", "AM", "BA", "CE", "DF", "ES", "GO", "MA", "MT", "MS", "MG", "PA", "PB", "PR", "PE", "PI", "RJ", "RN", "RS", "RO", "RR", "SC", "SP", "SE", "TO"}
-	for k := range values {
-		if groupName == values[k] {
-			estadual = true
+	var exists bool
+	jurisdicao := map[string]string{
+		"justica-eleitoral":    "Eleitoral",
+		"ministerios-publicos": "Ministério",
+		"justica-estadual":     "Estadual",
+		"justica-do-trabalho":  "Trabalho",
+		"justica-federal":      "Federal",
+		"justica-militar":      "Militar",
+		"justica-superior":     "Superior",
+		"conselhos-de-justica": "Conselho",
+	}
+
+	// Adaptando as URLs do site com o banco de dados
+	// Primeiro consultamos entre as chaves do mapa.
+	if jurisdicao[groupName] != "" {
+		groupName = jurisdicao[groupName]
+	} else {
+		// Caso não encontremos entre as chaves, verificamos entre os valores do mapa.
+		// Isso pois, até a consolidação ser finalizada, o front consulta a api com /Eleitoral, /Trabalho, etc.
+		for i := range jurisdicao {
```

Comment:
Essa consulta era para ser realizada entre os valores, certo? Se for essa a intenção, esse código não está certo. Deveria ser:

```go
for _, valor := range jurisdicao {
    if groupName == valor {

```